src/test/java/com/wikia/webdriver/elements/common/Navigate.java

```diff
@@ -19,6 +26,7 @@ public Navigate toPageByPath(String path) {
       Thread.sleep(2000);
     } catch (InterruptedException e) {
       e.printStackTrace();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)